x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_actions_column.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback } from 'react';
+import { EuiButtonIcon } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { LogEntryColumnContent } from './log_entry_column';
+import {
+  euiStyled,
+  ActionMenu,
+  Section,
+  SectionTitle,
+  SectionLinks,
+  SectionLink,
+} from '../../../../../observability/public';
+
+interface LogEntryActionsColumnProps {
+  isHovered: boolean;
+  isMenuOpen: boolean;
+  onOpenMenu: () => void;
+  onCloseMenu: () => void;
+  onViewDetails: () => void;
+}
+
+const MENU_LABEL = i18n.translate('xpack.infra.logEntryItemView.logEntryActionsMenuToolTip', {
+  defaultMessage: 'View Details',
```

Comment:
I wonder if this should be more generic as in "View actions"? Ultimately this menu will contain more than just the details link.

```suggestion
  defaultMessage: 'View actions',
```